storage/src/main/scala/coop/rchain/storage/regex/RegexPattern.scala

```diff
@@ -170,8 +195,172 @@ object CharClassPattern extends ParsedPattern {
   def apply(charSet: Set[Char], negateCharSet: Boolean): CharClassPattern =
     new CharClassPattern(charSet, negateCharSet)
 
-  def tryParse(str: CharSequence): Option[(CharClassPattern, Int)] =
-    throw new NotImplementedError("TODO")
+  private[this] val escapes =
+    Map('t' -> '\t', 'r' -> '\r', 'n' -> '\n', 'f' -> '\f', 'v' -> '\u0011')
+
+  private[this] val allSpecialChars = "\\\\[]|().?*+{}".toSet
+
+  //region predefined char classes
+  private[this] val wordCharClass: CharClassPattern = CharClassPattern(
+    "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz")
```

Comment:
Assign the strings here and below to constants and then use the constant twice instead of the string.  The two strings will never diverge because one is the negation of the other.

Reply:
Done.